### DISCUSSION TITLE
Is there any frontend demo or link to live project ?

### DISCUSSION THREAD

**Discussion by e-orlov** [datafeedr/wordpress-plugins#16] [Help] [answered]
Is there any fronend demo we could click through? Or maybe any user would share a link to an own project based on DF?

Particularly I would like to see, how look category and single product view.

**Comment by EricBusch** [comment, marked as answer]
Hi

We don't have an interactive admin demo but here's my own personal Datafeedr site https://www.geargrabber.net/ and here's one of customer's site (he gave us permission to share his URL) https://chubstr.com/.

Hope this helps!

If you have any other questions, please don't hesitate to ask.

Thanks,
Eric
